graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaBuilder.java

```diff
@@ -528,9 +520,43 @@ public GraphQLObjectType buildType(Table table) {
       }
     }
 
+    buildAggregationFunctions(builder);
+
     return builder.build();
   }
 
+  private void buildAggregationFunctions(GraphQLObjectType.Builder builder) {
+    builder.field(buildFunctionField(INT_FUNCTION, GraphQLInt));
+    // The GraphQLFloat corresponds to CQL double
+    builder.field(buildFunctionField(DOUBLE_FUNCTION, GraphQLFloat));
+    builder.field(buildFunctionField(BIGINT_FUNCTION, CqlScalar.BIGINT.getGraphqlType()));
+    builder.field(buildFunctionField(DECIMAL_FUNCTION, CqlScalar.DECIMAL.getGraphqlType()));
+    builder.field(buildFunctionField(VARINT_FUNCTION, CqlScalar.VARINT.getGraphqlType()));
+    builder.field(buildFunctionField(FLOAT_FUNCTION, CqlScalar.FLOAT.getGraphqlType()));
+    builder.field(buildFunctionField(SMALLINT_FUNCTION, CqlScalar.SMALLINT.getGraphqlType()));
+    builder.field(buildFunctionField(TINYINT_FUNCTION, CqlScalar.TINYINT.getGraphqlType()));
+  }
+
+  private GraphQLFieldDefinition buildFunctionField(
+      SupportedGraphqlFunction graphqlFunction, GraphQLScalarType returnType) {
+    return GraphQLFieldDefinition.newFieldDefinition()
+        .name(graphqlFunction.getName())
+        .description(
+            String.format("Invocation of an aggregate function that returns %s.", returnType))
+        .argument(
+            GraphQLArgument.newArgument()
+                .name("name")
+                .description("Name of the function to invoke")
+                .type(new GraphQLNonNull(GraphQLString)))
+        .argument(
+            GraphQLArgument.newArgument()
+                .name("args")
+                .description("Arguments passed to a function. It can be a list of column names.")
+                .type(new GraphQLList(new GraphQLNonNull(GraphQLString))))
```

Comment:
Since `AggregationsFetcherSupport.getAndValidateArgs` fails if this argument is missing, we can make it non-null as well:
```suggestion
                .type(new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLString)))))
```

If we support arg-less functions (like `now()`) in the future,  we might want to revisit this, but I'm fine with it since going from non-null to nullable is not breaking.